inbm/dispatcher-agent/dispatcher/xmlhandler.py

```diff
@@ -148,7 +148,7 @@ def get_attribute(self, xpath: str, attribute_name: str) -> str:
```

Comment:
Consider using a more robust XML parsing and manipulation library like `lxml` to handle XML operations. [medium]